test/test_mps.py

```diff
@@ -5884,6 +5861,98 @@ def test_view_all_dtypes_and_devices(self, device="mps"):
             x = torch.tensor([[1, 2], [3, 4], [5, 6]], dtype=dt, device=device)
             self.assertEqual(x.view(6).shape, [6])
 
+class TestConvolutionMPS(TestCase):
+    def test_conv1d_all_strides_paddings(self):
+        # https://github.com/pytorch/pytorch/issues/82921
+        def helper(stride, padding):
+            y_cpu = torch.randn(1, 57, 40)
+            conv_cpu = nn.Conv1d(57, 20, stride=stride, padding=padding, kernel_size=3, bias=False)
```

Comment:
nit: why such big channels sizes?

Reply:
Ah this is from the issue filed; https://github.com/pytorch/pytorch/issues/82921

Reply:
I would be careful in general with the size of the test sample. You want to keep your tests as fast as possible!